core/textile/sharing.go

```diff
@@ -3,22 +3,24 @@ package textile
 import (
 	"context"
 
+	"github.com/FleekHQ/space-daemon/core/space/domain"
 	crypto "github.com/libp2p/go-libp2p-crypto"
 )
 
-func (tc *textileClient) ShareFilesViaPublicKey(ctx context.Context, bucketName string, paths []string, pubkeys []crypto.PubKey) error {
+func (tc *textileClient) ShareFilesViaPublicKey(ctx context.Context, paths []domain.FullPath, pubkeys []crypto.PubKey) error {
 	// TOOD: uncomment once release and upgraded txl pkg
-	// ctx := tc.getHubCtx(ctx)
+	// ctx = tc.getHubCtx(ctx)
 
 	// for _, pth := range paths {
-	// 	var roles map[string]buckets.Role
-	// 	for _, pk := range pubkeys {
-	// 		roles[pk] = buckets.Role.Writer
-	// 	}
-	// 	err := tc.EditPathAccessRoles(ctx, key, pth, roles)
-	// 	if err != nil {
-	// 		return err
-	// 	}
+	//   ctx = tc.GetRemoteBucketContext(ctx, pth.DbId, pth.Bucket)
```

Comment:
@maurycy this is another one I wanted to double check if there is overlap before I go ahead and create one.